### DISCUSSION TITLE
Integrate coc status with lualine

### DISCUSSION THREAD

**Discussion by sigmaSd** [nvim-lualine/lualine.nvim#731] [Q&A] [answered]
If I run a typescipt file I get `Deno version` at the start of the line, is there a way to turn it off, I personally don't like it and also it seems to be missing some padding.

![image](https://user-images.githubusercontent.com/22427111/178106104-5dabfb56-d5e4-4a74-a0b5-aeeb00ca69ce.png)

**Comment by sigmaSd** [comment]
I guess I get other info in that same place, but I prefer to disable it if possible.

**Comment by shadmansaleh** [comment, marked as answer]
That's not lualine . Some other plugin is displaying that . Find which plugin it is and if it provides a way to disable it . IIRC I've seen similar issues previously with coc.nvim .

**Reply by sigmaSd** [reply]
Thanks

**Comment by sigmaSd** [comment]
It seems to be cause by `"set statusline^=%{coc#status()}%{get(b:,'coc_current_function','')}`

would you have an example on how can I nicely integrate this back to lualine @shadmansaleh ?

**Reply by shadmansaleh** [reply]
just put them in lualines config
```lua
lualine_c = {'coc#status', 'b:coc_current_function'}
```

**Reply by sigmaSd** [reply]
thanks again!
